src/redux/services/charts/index.js

```diff
@@ -0,0 +1,127 @@
+export default function resolveChartsData() {
+    return new Promise((resolve) => {
+        setTimeout(() => {
+            resolve({
+                chartsData: {
```

Comment:
here resolve the object directly, not need of chartsData key and put all data in a const above the function,
also I think we only need to resolve here the chart values, not data like title, type, colors,
then pass only labels and values, look at dashboard services,
the other data is set fixed in the view